style.css

```diff
@@ -0,0 +1,361 @@
+@import url('https://fonts.googleapis.com/css2?family=Comforter&family=Oswald:wght@300&family=Quintessential&family=Raleway:wght@300&family=Ubuntu:wght@300&display=swap');
+
+:root {
+    --font-comforter: 'Comforter', cursive;
+    --font-oswald: 'Oswald', sans-serif;
+    --font-quintessential: 'Quintessential', cursive;
+    --font-raleway: 'Raleway', sans-serif;
+    --font-ubuntu: 'Ubuntu', sans-serif;
+    --primary-color: #00ADB5;
+    --secondary-color: #393E46;
+    --dark-color: #222831;
+    --light-color: #EEEEEE;
+    --badge-color: #FF5722;
+    --black-color: #000;
+    --white-color: #fff;
+    --default-transition: all 0.2s ease-in;
+}
+
+* {
+    box-sizing: border-box;
+    margin: 0;
+    padding: 0;
+    font-family: var(--font-ubuntu);
+}
+
+html {
+    font-size: 100%;
+    scroll-behavior: smooth;
+}
+
+body {
+    background: var(--light-color);
+}
+
+img {
+    width: 100%;
+    height: 100%;
+    object-fit: contain;
```

Comment:
Try to play with object-cover and object-fil for the homepage main slider.

Reply:
Okay nice I was thinking what to do to make it cover tha whole with and look good at the same time.